src/services/sam/monitors/types/monitorResponseTypes.ts

```diff
@@ -0,0 +1,30 @@
+export interface GetMonitorServicesResponse {
+  name: string,
+  key: string,
+  version: string,
+  summary: string,
+  authors: Array<string>,
+  author_emails: Array<string>,
+  uuid: string,
+  hidden_stages: Array<string>,
+  created_at: string,
+  updated_at: string
+}
+
+
+export interface Kpi {
+  uuid: string,
+  service_uuid: string,
+  // within the db is just JSON
+  kpis: any
```

Comment:
Don't use `any`. It might be in the DB as JSON, but I believe that there are pydantic models that represent this data structure that you can use as a guide. Even there aren't pydantic models, we should still correctly reflect the structure here.